README.md

```diff
@@ -1,38 +1,51 @@
-# ![nf-core/blobtoolkit](docs/images/nf-core-blobtoolkit_logo_light.png#gh-light-mode-only) ![nf-core/blobtoolkit](docs/images/nf-core-blobtoolkit_logo_dark.png#gh-dark-mode-only)
+# ![sanger-tol/blobtoolkit](docs/images/nf-core-blobtoolkit_logo_light.png#gh-light-mode-only) ![sanger-tol/blobtoolkit](docs/images/nf-core-blobtoolkit_logo_dark.png#gh-dark-mode-only)
 
-[![GitHub Actions CI Status](https://github.com/nf-core/blobtoolkit/workflows/nf-core%20CI/badge.svg)](https://github.com/nf-core/blobtoolkit/actions?query=workflow%3A%22nf-core+CI%22)
-[![GitHub Actions Linting Status](https://github.com/nf-core/blobtoolkit/workflows/nf-core%20linting/badge.svg)](https://github.com/nf-core/blobtoolkit/actions?query=workflow%3A%22nf-core+linting%22)
+[![GitHub Actions CI Status](https://github.com/sanger-tol/blobtoolkit/workflows/nf-core%20CI/badge.svg)](https://github.com/sanger-tol/blobtoolkit/actions?query=workflow%3A%22nf-core+CI%22)
+[![GitHub Actions Linting Status](https://github.com/sanger-tol/blobtoolkit/workflows/nf-core%20linting/badge.svg)](https://github.com/sanger-tol/blobtoolkit/actions?query=workflow%3A%22nf-core+linting%22)
 [![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?logo=Amazon%20AWS)](https://nf-co.re/blobtoolkit/results)
 [![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8)](https://doi.org/10.5281/zenodo.XXXXXXX)
 
 [![Nextflow](https://img.shields.io/badge/nextflow%20DSL2-%E2%89%A521.10.3-23aa62.svg)](https://www.nextflow.io/)
 [![run with conda](http://img.shields.io/badge/run%20with-conda-3EB049?logo=anaconda)](https://docs.conda.io/en/latest/)
 [![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?logo=docker)](https://www.docker.com/)
 [![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg)](https://sylabs.io/docs/)
-[![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/nf-core/blobtoolkit)
+[![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/sanger-tol/blobtoolkit)
 
 [![Get help on Slack](http://img.shields.io/badge/slack-nf--core%20%23blobtoolkit-4A154B?logo=slack)](https://nfcore.slack.com/channels/blobtoolkit)
 [![Follow on Twitter](http://img.shields.io/badge/twitter-%40nf__core-1DA1F2?logo=twitter)](https://twitter.com/nf_core)
 [![Watch on YouTube](http://img.shields.io/badge/youtube-nf--core-FF0000?logo=youtube)](https://www.youtube.com/c/nf-core)
 
```

Comment:
Either remove Slack, Twitter and YouTube or update it to the correct links. It is all pointing to `nf-core` at this point.